recipes/emsarray/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set name = "emsarray" %}
+{% set version = "0.1.0" %}
+{% set build_num = 0 %}
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/emsarray-{{ version }}.tar.gz
+  sha256: 63c92c95483ec263aecf88266acf102513e2ce74099c035172affe4eff959729
+
+build:
+  number: {{ build_num }}
+
+outputs:
+  - name: emsarray-core
+    build:
+      noarch: python
+      entry_points:
+        - emsarray = emsarray.cli:main
+    script: ./build_base.sh
```

```suggestion
    script: build_base.sh
```
Just to make it simpler.